cache/test_cache.py

```diff
@@ -1713,6 +1713,12 @@ def test_h2(self):
         self.assertEqual(len(srv.requests), 1, "Request should be taken from srv_front")
         self.assertEqual(response, "bar")
 
+        srv = self.get_server("srv_front")
+        client = self.get_client("front-1")
+        response = self.get_response(client)
```

Comment:
Sorry, I don't understand why this request is necessary? How does it differ from the next one?

Reply:
Evgenii checks that Tempesta didn't override cached response, it may happens during transfer the first cached response. To do this we must do 3 requests. 1. Not from cache. 2. From cache, here cache may be corrupted.  3. From cache, here check validity.